### PR TITLE
Av starting on UI

### PR DESCRIPTION
**Pull request by AnnaVi11arrea1**

### CODE REVIEW

**Review by dgarcia13-99 — APPROVED** [review]
looks good!

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```
Comment (aldoportillo) [inline app/controllers/application_controller.rb:2]: nice

File: app/views/comments/index.html.erb [hunk 2]
```diff
@@ -1,5 +1,3 @@
-<p style="color: green"><%= notice %></p>
-
```
Comment (aldoportillo) [inline app/views/comments/index.html.erb:2]: good

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -15,9 +16,18 @@
...
   <body>
-    <%= render "shared/navbar" %>
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
-    <%= yield %>
-  </body>
-</html>
+    <%= render partial: "shared/navbar" %>
+
+    <div class="container">
```
Comment (aldoportillo) [inline app/views/layouts/application.html.erb:21]: fix indentation

File: config/routes.rb [hunk 4]
```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
+   root "photos#index"
```
Comment (aldoportillo) [inline config/routes.rb:2]: nice

**Review by aldoportillo — CHANGES_REQUESTED** [review]
Looks good just fix indentation and let me know when. No need t resubmit URL.

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (AnnaVi11arrea1) [reply app/views/layouts/application.html.erb:21]: indentation fixed. 
Screenshot 2024-10-16 at 4 29 50 PM

**Pull request merged**